run.sh

```diff
@@ -85,6 +85,24 @@ function setup() {
       run_scripts $BINDER "create.sh"
     popd
     run_scripts "redis" "create.sh"
+    if [ "$PLATFORM" == "cloudfoundry" ];
```

Comment:
Since the CF config server service instance requires more time to get up and running, we may not want this as part of the `setup`. Hence, I have this service running for CI environment and we could still create using the create scripts above.